sqlalchemy_api_handler/bases/accessor.py

```diff
@@ -9,25 +9,21 @@ def get(self,
             path,
             with_get_path_error=True):
 
```

Comment:
En fait je pense que le plus propre c'est de en plus garder la guard close comme t'avais fais au début du style :
```suggestion
if path is None or path ==  '':
    return None
```

Reply:
si on met que ça retourne None pour path == '', ca casse le test test_get_with_an_index_path_at_entity dans ce cas....

Reply:
même si tu laisses le code plus bas ? :
if child_path == '':
                return value

Reply:
ha oui ok dans ce cas yes

Reply:
ok:)